src/arc_utilities/reliable_tf.py

```diff
@@ -0,0 +1,113 @@
+from functools import partial
+from threading import Thread, Event
+
+import numpy as np
+
+import ros_numpy
+import rospy
+import tf
+from arc_utilities.tf2wrapper import TF2Wrapper
+from geometry_msgs.msg import Pose
+from rospy import Rate
+
+
+def tfkey(*, parent, child):
+    k = (parent, child)
+    return k
+
+
+def send_transform(exit_event, tfw, translation, quaternion, parent, child, is_static):
+    r = Rate(10)
```

Comment:
Not enjoying the duplication here; thoughts on having a higher level
```python
def send_at_rate(exit_event, sender, rate=10):
   r = Rate(rate)
   while not exit_event.is_set():
      sender()
      r.sleep()
```
still results in repeating the arguments though...

Reply:
I made an attempt to reduce duplication, it's a little hard to read though...